### ISSUE TITLE
Change volume using iPhone volume buttons in control centre remote

### ISSUE THREAD

**Issue by MED1975** [merdok/homebridge-webos-tv#499]
Hi. Recently I have been unable to use my phone's volume buttons when using the CC remote to raise or lower the volume. Is this function no longer working is iOS 17 or is something else wrong with my configuration?

Thanks!

**Comment by merdok** [comment]
Still working fine. I see no issues with that and I am on iOS17. So it is probably something on your side...

**Comment by MED1975** [comment]
Thanks - I've gone over my config file and cannot see anything obvious. You don't have to set anything specific there for this to work I believe?

**Comment by merdok** [comment]
Yeah, there is no need for any config modifications. It should work out of the box!

**Comment by MED1975** [comment]
Hi - I have now got this to work. I have been turning on my TV (and auto switching to the Apple TV input) using the ATV physical remote. When I have done it this way I could not change volume from my phone using the CC volume buttons.

However - it has started working after I changed the input to the ATV using the input selector for the TV in the home app. Once this has been done I could change volume using my phone from the CC.
